frontend/src/component/Header/Logo.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
 
 const Logo = props => {
   return (
-    <a className='header__logo navbar-brand' onClick={props.onClickImg}>
+    <Link className='header__logo navbar-brand' to={props.to}>
       <img className='header__logo__img' src={props.logoSrc} />
-    </a>
+    </Link>
   )
 }
 export default Logo
 
 Logo.propTypes = {
   logoSrc: PropTypes.string.isRequired,
-  onClickImg: PropTypes.func.isRequired
+  to: PropTypes.string
```

Comment:
for each non required props, you should have a defaultProps value.
Also check that the default props works, meanings clicking on the logo without having a `to` specified to `<Logo />` doesn't generate a js error